internal/database/mysql.go

```diff
@@ -138,16 +141,23 @@ func (db *MySQLDatabase) getOpenID() int {
 }
 
 func (db *MySQLDatabase) AddUser(username string, email string, password string) error {
+	hashedPassword, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
+	log.Printf("Hashed Password: %s, from User Password: %s", hashedPassword, password)
```

Comment:
Again, I don't think we should clutter our server logs with hashed user passwords.

Reply:
Apologies left this in from testing.